addons/pager/jquery.tablesorter.pager.js

```diff
@@ -328,7 +328,11 @@
 				if (p.fixedHeight && !p.isDisabled) {
 					h = $.data(table, 'pagerSavedHeight');
 					if (h) {
-						d = h - $b.height();
+						var bs = 0;
+						if($(table).css('border-spacing').split(" ").length>0){
```

Comment:
This `if` statement *should* always be `true`.... is there a particular browser where it isn't? When I use `$('table').css('border-spacing')` on a table without any css changes, I will see `2px 2px`.